packages/arb-token-bridge-ui/src/components/common/HeaderMenu.tsx

```diff
@@ -20,8 +21,12 @@ export function HeaderMenuDesktop(
   return (
     <Popover as="div" className="relative inline-block text-left">
       <div>
-        <Popover.Button className="arb-hover hidden rounded-md text-base text-white lg:inline-flex">
+        <Popover.Button className="arb-hover hidden items-center rounded-md text-base text-white lg:inline-flex lg:p-1">
```

Comment:
need to change the total size of (padding + margin) from 40px to 30px on desktop according to spec
> Text is spaced 30px away from each text element (used to be 40px)

Reply:
30px seems a little hard to achieve with even-numbered spacing and padding unless we hard-code odd values (which I don't prefer tbh). 
So for now I have squeezed it from 40px to 32px. (Margin - 24px + Padding left : 4px + Padding right : 4px) and it looks right.
Let me know if this passes else I will look deeper into this.

Reply:
Yea, that makes sense. Can you leave a comment on the spec to let Val know why it's 32px? We can also align the size scale from now on (multiple of 4px/16px etc)